sensor_msgs/msg/CompressedImage.msg

```diff
@@ -1,13 +1,16 @@
 # This message contains a compressed image.
 
-std_msgs/Header header # Header timestamp should be acquisition time of image
+std_msgs/Header header       # Header timestamp should be acquisition time of image
                              # Header frame_id should be optical frame of camera
                              # origin of frame should be optical center of cameara
                              # +x should point to the right in the image
                              # +y should point down in the image
                              # +z should point into to plane of the image
 
-string format                # Specifies the format of the data
+uint32 height                # image height, that is, number of rows
+uint32 width                 # image width, that is, number of columns
```

Comment:
Prior to this PR, how are users of this message determining this?

Reply:
As this was encoded into the payload before, we need some more semantics about when it should be used and when not. It's not great to have data that can potentially disagree.